odd-collector/odd_collector/adapters/ckan/tests/conftest.py

```diff
@@ -0,0 +1,291 @@
+import pytest
+from oddrn_generator import CKANGenerator
+
+from odd_collector.domain.plugin import CKANPlugin
+from odd_collector.adapters.ckan.mappers.models import (
+    Group,
+    Organization,
+    Dataset,
+    Resource,
+)
+
+
+@pytest.fixture
+def ckan_adapter_config(request):
```

Comment:
**nit**: Naming is debatable thing but it's not an config dict/dataclass/object will be returned but a callable function. So if we want to be semantically correct the fixture should be named with a noun or maybe even with `_factory` prefix.
```suggestion
def create_ckan_adapter_config(request): # or ckan_adapter_config_factory
```

Reply:
This is appliable to all other fixutres below.

Reply:
makes sense, done.